lib/generators/boring/brakeman/install/install_generator.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+module Boring
+  module Brakeman
+    class InstallGenerator < Rails::Generators::Base
+      desc 'Adds Brakeman to the application'
+      source_root File.expand_path('templates', __dir__)
+
+      RUBY_VERSION_FILE = '.ruby-version'
+
+      DEFAULT_RUBY_VERSION = '.ruby-version'
+
+      class_option :skip_config, type: :boolean, aliases: '-c',
+                                                desc: 'Skip adding Brakeman configuration. Defaults to false'
+
+      class_option :github, type: :boolean, aliases: '-gh',
```

Comment:
Let's remove github and gitlab option and use "ci" instead which should take enum [github, gitlab] and should be empty by default